QUARKUS-5615.md

```diff
@@ -0,0 +1,30 @@
+## Links to the resources
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5615
+- Upstream documentation PR: https://github.com/quarkusio/quarkus/pull/46182
+- Follow-up to the original PR: https://github.com/quarkusio/quarkus/pull/46211
+- Serialisation PR: https://github.com/quarkusio/quarkus/pull/41063
+- Serialisation documentation PR: https://github.com/quarkusio/quarkus/pull/42289
+- Related guide: https://quarkus.io/guides/rest#reflection-free-jackson-serialization
+
+## Scope of the testing 
+This feature adds a new Quarkus property (`quarkus.rest.jackson.optimization.enable-reflection-free-serializers`) which should not affect behavior of Quarkus.
+If this property is activated for a project which uses Jackson for rest endpoints, but doesn't use advanced Jackson annotations (anything, besides @JsonProperty and @JsonIgnore), then Quarkus will use compile-time (de)serialisation, which is a bit faster, than the runtime-one.
+
+## Getting familiar with the feature
+- Read the guides:
+ * https://quarkus.io/guides/rest#reflection-free-jackson-serialization
+ * https://quarkus.io/blog/quarkus-metaprogramming/
+
+
+## Community status of the extension/feature:
+Affected extensions (REST and Jackson) are supported, but the feature itself is in Technology Preview.
```

Comment:
Technology Preview status is planned for product, you are mixing it here with community. Please adjust the wording.

Reply:
I took the info from upstream guide: https://github.com/quarkusio/quarkus/pull/46182/files

Upstream guide describes community release, no?

Reply:
ok, upstream usually uses just `Preview` keyword and product `Technology Preview` as it's a thing defined by RH policy https://access.redhat.com/support/offerings/techpreview